### ISSUE TITLE
[ALM Accelerator - QUESTION] Deployment profile with only 2 environments

### ISSUE THREAD

**Issue by dspencelink** [microsoft/coe-starter-kit#7522]
### What is your question?

Hello.  We've been using ALM Accelerator for many months now in a 3 environment setting: Dev --> Test --> Prod and it was fairly straightforward to set up.  I now have a new scenario where we just have a Dev environment and a Prod environment.  I've set up ALM accelerator and the 2 environments but what do I need to do to ensure that I just have Dev and Prod pipelines in the deployment profile?  Do I need to manually edit YAML files?  If so, what should I do?  Sorry, i'm a bit of a noob here.  Thanks.

### What component are you experiencing the issue with?

ALM Accelerator Canvas App

### What solution version are you using?

1.0.20231109.3

AB#2115

**Comment by dspencelink** [comment]
Does anyone have familiarity with doing this?  Do I just set the Validation and Test environments as 'Not Deployable', then set the Production trigger to be when a specific branch is updated?  TIA

**Comment by v-bjaiswal** [comment]
Hi @dspencelink , if you want to go with the 2 environments then you can do that by 2 methods.

1. By using the "Import Solution" button in ALM canvas app. Commit the solution in dev and then you can use the specific environment where you want the deployment.
![image](https://github.com/microsoft/coe-starter-kit/assets/102227309/c1b0036f-e345-4d80-b3da-9f7af357b251)

2. Other way to to ensure that you have committed your changes into Production pipeline. As this pipeline will trigger as soon as something changes on main branch.
![image](https://github.com/microsoft/coe-starter-kit/assets/102227309/19e8b38e-2acb-4f4c-8c4a-844dfdd6ecd9)

**Comment by dspencelink** [comment]
Thanks @v-bjaiswal.  I've committed, then deployed to Production, which deploys to main but it didn't trigger the pipeline to deploy the Solution to Production:
![image](https://github.com/microsoft/coe-starter-kit/assets/133893716/c89c1015-daf0-4fb1-81b3-5c1a80f4d098)

If I go into the Pipelines and manually run the Production pipeline, it successfully deploys the solution.  I just need to figure out how to trigger it because it's not triggering from ALM Accelerator.

![image](https://github.com/microsoft/coe-starter-kit/assets/133893716/8562181b-5832-47a4-8c7d-52d1f9a5f181)

**Comment by dspencelink** [comment]
I figured it out.  My Solution is named EntReporting.  After I commit, when I deploy to Prod, it tries to copy from the EntReporting branch to Main but there's nothing in that branch because I haven't deployed to Test.  So I found if I deploy to Test first, then it copies from the 'Developername-Solution' branch into the EntReporting branch.  Then when I deploy to Prod, it pulls from EntReporting branch into Main and deploys to Prod.  So I just need to deploy to Test, then deploy to Prod.
